recipes/head_node_slurm_finalize.rb

```diff
@@ -49,14 +49,14 @@
     require 'shellwords'
 
     # Example output for sinfo
-    # $ /opt/slurm/bin/sinfo -N -h -o '%N %t'
+    # $ #{node['cfncluster']['slurm']['install_dir']}/bin/sinfo -N -h -o '%N %t'
     # ondemand-dy-c5.2xlarge-1 idle~
     # ondemand-dy-c5.2xlarge-2 idle~
     # spot-dy-c5.xlarge-1 idle~
     # spot-st-t2.large-1 down
     # spot-st-t2.large-2 idle
     is_fleet_ready_command = Shellwords.escape(
-      "set -o pipefail && /opt/slurm/bin/sinfo -N -h -o '%N %t' | { grep -E '^[a-z0-9\\-]+\\-st\\-[a-z0-9]+\\-[0-9]+ .*' || true; } | { grep -v -E '(idle|alloc|mix)$' || true; }"
+      "set -o pipefail && #{node['cfncluster']['slurm']['install_dir']}/bin/sinfo -N -h -o '%N %t' | { grep -E '^[a-z0-9\\-]+\\-st\\-[a-z0-9]+\\-[0-9]+ .*' || true; } | { grep -v -E '(idle|alloc|mix)$' || true; }"
```

Comment:
This line is too long: C: Layout/LineLength: Line is too long. [213/200]
Could you break it into two lines with `\`?